src/routes/admin/ciraconfig/ciraValidator.ts

```diff
@@ -46,11 +46,38 @@ export const ciraInsertValidator = (): any => {
       .matches('^[a-zA-Z0-9]+$') // As per AMT SDK, accepts only alphanumeric values
       .withMessage('MPS user name should be alphanumeric'),
```

Comment:
may want to check the username isn't more than 16 chars.

Reply:
done